Gems/ImGui/Code/Source/ImGuiManager.cpp

```diff
@@ -724,8 +724,16 @@ void ImGuiManager::ToggleThroughImGuiVisibleState(int controllerIndex)
     }
 
     m_menuBarStatusChanged = true;
+    ImGuiManagerNotificationBus::Broadcast(
+        &IImGuiManagerNotifications::ImGuiSetEnabled, m_clientMenuBarState == DisplayState::Hidden);
```

Comment:
If the ImGuiManagerNotificationBus is removed and replaced with a single ImGuiSetEnabled event.
Then this call can be updated to be
`m_setEnabledEvent.Signal(m_clientMenuBarState == DisplayState::Hidden);`

Reply:
Changed made.